packages/suite/src/actions/wallet/send/sendFormThunks.ts

```diff
@@ -36,7 +36,7 @@ import { findLabelsToBeMovedOrDeleted, moveLabelsForRbfAction } from '../moveLab
 export const MODULE_PREFIX = '@send';
 
 export const saveSendFormDraftThunk = createThunk(
-    `${MODULE_PREFIX}/saveSendFormDraftThunk`,
+    `${MODULE_PREFIX}/ `,
```

Comment:
This is probably not intentional

Reply:
27dab3bed715b87cd0b1ec77eb818a99c9355c23